exercism/website
# Add internal links on track dashboard for main toolings - WIP

- Add route
- Setup css
- Add lhs: top header, stats, rhs: volunteers
- Add tooling-status variants, track-status section
- Fix border on status icon, add gradient section
- Rename svgs
- Add gradient svgs
- Add cool stuff
- Complete lhs of dashboard
- Add header background, fix highlighting, add new gradient bg variant
- Make tooling status indicator code look cooler
- Add all the icons
- fix line height, add pointer cursor to %details
- Remove no-effect bg color
- Fix typo in comment
- Add random numbers to closure
- Format long numbers
- Add some real data
- Play more with highlight background, then remove all of it
- Add more dynamic data
- Remove extra text
- Add map
- Create AvatarGroup component, Credits component
- Add contributors data
- Add credits component
- Add contributors grid
- Update jest config
- Fix typescript error
- One more tiny tsconfig fix
- Make classnames local in css
- Remove redundant & from css
- Add more props to credits ruby component. Fix type of Approaches in application.tsx
- Fix type of impact chart
- Cheery pick credits ruby VC
- Add new credits components
- Add tooltips, change some wording (#3136)
- Move track build page to separate controller
- Setup infrastructure for caching build status
- Add tests for creating and updating build status in redis
- Use symbolized keys for build status tests
- Populate students section
- Populate submissions section
- Refactor average number calculation to helper method
- Populate mentor_discussions section
- Populate build section
- Populate concepts section
- Working on num_students_learnt
- Also set completed_at for :published solution factory trait
- Add num_learnt field to concepts
- Add concept_exercises field
- Add practice_exercises field
- Combine syllabus fields
- Add job to update all tracks' build status
- Add test runner section
- Add percentages to test runner data
- Add syllabus volunteers
- add method to serialize volunteers
- Add volunteers section
- Add volunteers section
- Add test runner volunteers
- Add representer volunteers
- Add analyzer volunteers
- Add analyzer display rate percentage
- Add num representations
- Add more representer information
- Add num_comments'
- Move build status logic into separate module
- Add num_comments for analyzer
- Add TODO
- Populate some data
- Even more populated data
- Add syllabus health
- Add analyzer health
- Add scope for analysis with comments
- Allow creating submission analysis with comment via factory
- Fix incorrect analyzer display rate percentage
- Add representer health
- Use OpenStruct instead of Hash for build status
- Add practice exercises health
- Allow fetching prob-specs data
- Access foregone exercises
- Determine unimplemented exercises
- Ensure build status uses up-to-date prob-specs repo
- Add target for practice exercises
- Add target for concept exercises
- Use correct tooltip values
- Start out with test_runner health
- Use build status via filter
- Add health todos
- Add stub test for test runner health
- Revert change
- Determine test runner health
- Add links to build page
- Correctly pluralize texts
- Add goal for syllabus
- Add test runner goal
- Refactor
- Use users
- Change health status
- Add ids
